google/cloud/forseti/scanner/scanners/config_validator_util/validator_client.py

```diff
@@ -105,37 +106,23 @@ def paged_audit(self, assets):
         """
         paged_assets = []
         current_page_size = 0
-        data_loaded = 0
         for asset in assets:
             asset_size = sys.getsizeof(asset)
             # Dictionary size is not properly reflected, cast the dictionary to
             # string instead to estimate the actual dictionary size.
             asset_size += sys.getsizeof(str(asset.resource))
             asset_size += sys.getsizeof(str(asset.iam_policy))
-            if current_page_size + asset_size >= self.max_page_size:
-                LOGGER.debug('Adding paged data to Config Validator, size: '
-                             '%s, content: %s',
-                             current_page_size, paged_assets)
-                data_loaded += current_page_size
-                self.add_data(paged_assets)
-                paged_assets = []
-                current_page_size = 0
-            if data_loaded >= self.max_audit_size:
-                LOGGER.debug('Auditing data, size: %s', data_loaded)
-                violations = self.audit()
-                self.reset()
-                data_loaded = 0
+            if current_page_size + asset_size >= self.max_audit_size:
+                violations = self.review(paged_assets)
                 if violations:
                     yield violations
+                paged_assets = []
```

Comment:
Should paged_assets = [] and current_page_size = 0 be unindented once?

Reply:
That looks like the right indentation to me, per audit we want to remove all the audited data and reset the page size hence it's inside the if statement.

Reply:
Ok! Thanks for the clarification.